src/PhpSpec/Formatter/JsonFormatter.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Chris Kruining <chrise@keruining.eu>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace PhpSpec\Formatter
+{
```

Comment:
don't use the curly braces for the namespace. This forces to indent the whole file for nothing.